src/Microsoft.DotNet.Wpf/specs/tooltip.md

```diff
@@ -0,0 +1,305 @@
+Tooltip behavior
+==
+This document describes the behavior of tooltips, 
+especially the changes needed to meet the accessibility requirements of WCAG 2.1.
+
+
+# Background
+
+Microsoft is commmitted to making its products and UI platforms meet the accessibility requirements of WCAG 2.1 in a uniform way.
+This document describes how WPF's tooltips will honor that commitment.
+
+The document includes the following sections:
+- **Requirements**.  Links to sources of information that drove the decisions described here.
+- **Applicability**.  How the new behavior will be delivered.
+- **Conceptual pages**.  Specifies tooltip behavior.  The emphasis is on new behavior, but old behavior is included when helpful for context.  This is the public-facing spec, as it might appear in "overview" or "how-to" pages on docs.microsoft.com. 
+- **API Pages**.  Public-facing spec for new APIs, as it might appear in "reference" pages on docs.microsoft.com.
+- **Implementation**.  Internal-facing spec - details needed by implementers, testers, and reviewers, but not part of the public documentation.
+- **Discussion**.  Remarks that aren't part of the spec, but help understand it.  Including how the spec meets the requirements, goals and non-goals, alternatives that were considered, etc.  
+- **Summary of behavior changes**.  Consolidated in one place for convenience.
+- **API Details**.  Consolidated in one place for convenience.
+
+
+# Requirements 
+This spec cites the following sources (the [brackets] contain abbreviations used elsewhere in this document):
+
+1. WCAG 2.1, section 1.4.13: [Content on hover or focus](https://www.w3.org/TR/WCAG21/#content-on-hover-or-focus) [WCAG] 
+2. WCAG 2.1:  [Understanding 1.4.13](https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html)   [WCAG]
+3. MAS 1.4.13:  [Content on hover or focus](https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7B56729308-859e-4b32-ab27-04d4364b3003%7D&action=view&wdLOR=c9C9640FB-F04E-401A-BF46-A79887F322D3&wdAccPdf=0&wdparaid=38854747) [MAS]
+4. The [Figma diagram](https://www.figma.com/file/8LeT0Eq7a3ylAyM19c0FJp/ASC%3A-Tooltip?node-id=110%3A0) (as of February 2021) [FD]
+5. Email thread (MS internal) [ET] 
+6. WPF [Tooltip Overview](https://docs.microsoft.com/en-us/dotnet/desktop/wpf/controls/tooltip-overview?view=netframeworkdesktop-4.8) (and related links) [TO]
```

Comment:
What does "[TO]" mean?

Reply:
It is the "abbreviation used elsewhere in this document" (see line 24) to refer to this link.
The letters TO stand for Tooltip Overview.